content/md/franklin-insights/franklin-insights-subscriptions.md

```diff
@@ -0,0 +1,47 @@
+---
+id: franklin-insights-subscriptions
+themes: franklin-insights
+title: Franklin Insights **subscriptions**
+ee-only: true
+related: franklin-insights-getting-started, franklin-insights-mappings
+---
+
+# Overview
+
+Once you [activate the Franklin connection](/franklin-insights-getting-started.html#franklin-activation) and perform the [mapping of Franklin identifiers](/franklin-insights-mappings.html#identifiers-mapping), you have everything you need in order to **subscribe products to Franklin for automatic enrichment**.
+
+When you subscribe a product to Franklin, Franklin **collects and validates information on demand** from hundreds of sources and its internal database, and then utilizes **Big Data, Artificial Intelligence, and Machine Learning** in order to provide you with **rich, accurate, and structured product information suggestions**.
+
+# How to subscribe products
+
+You have 2 ways to subscribe products to Franklin:
+
+- Via the **Product Edit Form**
+
+![image](../img/Akeneo_Screenshot_Franklin_Insights_PEF_Subscription.png)
+
+- Via **bulk action**
+
+![image](../img/Akeneo_Screenshot_Franklin_Insights_Bulk_Actions.png)
+
+In order for you to receive attribute values proposals for subscribed and enriched products, you need to **match between Franklin’s attributes and your catalog attributes** in each family using the [Franklin attributes mapping UI](/franklin-insights-mappings.html#identifiers-mapping).
```

Comment:
we should explain that users won't get suggested data instantly and why. We also miss the notification related to the unmapped attributes